infra/dockerfile.commit_retrieval

```diff
@@ -4,18 +4,18 @@ COPY infra/git_hyper_blame_patch /root/git_hyper_blame_patch
 
 # Mercurial need Python2 :(
 # git and git-hyper-blame (from depot_tools) are required by the annotate pipeline.
-RUN apt-get update && \
-    apt-get install -y python python-pip git curl && \
+RUN apk update && apk upgrade && \
+    apk add --no-cache gcc g++ libgomp lapack-dev git libexecinfo-dev make libc6-compat && \
+    apk add python-dev py-pip curl && \
```

Comment:
Why a new line to add these? You can add them together with the other packages above

Reply:
Or at least you have to use ` --no-cache` here too

Reply:
Will do. Thanks